operations/power_management/Save_Management_Network_Switch_Configurations.md

```diff
@@ -4,69 +4,36 @@ Switches must be powered on and operating. This procedure is optional if switch
 
 **Optional Task:** Save management network switch configurations before removing power from cabinets or the CDU. Management switch names are listed in the `/etc/hosts` file.
 
-## Obtain the list of switches
-
-From the command line on any NCN run:
-
-```bash
-grep 'sw-' /etc/hosts
-```
-
-Example output:
-
-```bash
-10.254.0.2 sw-spine-001
-10.254.0.3 sw-spine-002
-10.254.0.4 sw-leaf-001
-ncn-m001:~ #
-```
-
 ## Save switch configs
 
 ### Aruba Switch and HPE Server Systems
 
 On Aruba-based systems all management network switches will be Aruba and the following procedure.
 For each switch:
 
-1. `ssh` to the switch
+1. Run the command below
```

Comment:
For both of these sections, I think it would flow better to include the example command under this step instead of after all of the steps.